src/components/Person/Person.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+function infoAboutPartner(sex, partnerName) {
+  if (sex === 'm') {
+    return `My wife's name is ${partnerName}`;
+  }
+
+  return `My husband's name is ${partnerName}`;
+}
+
+function Person({ name, age, sex, isMarried, partnerName }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {(age !== 0) && (<p className="Person__age">{`I am ${age}`}</p>)}
```

Comment:
Let's make this core more understandable
```suggestion
      {age && (
      	<p className="Person__age">
      		{`I am ${age}`}
      	</p>
      )}
```